### DISCUSSION TITLE
Ternary Operation To Toggle Window.isBorderless To True Or False?

### DISCUSSION THREAD

**Discussion by BugInfestedCode** [MonoGame/MonoGame#8605] [General]
Hello,

Based upon RB Whittaker's keyboard input tutorial I originally had the following code in my project which works as expected.

```c#
private KeyboardState oldState;

KeyboardState newState = Keyboard.GetState();

if(oldState.IsKeyUp(Keys.Enter) && newState.IsKeyDown(Keys.Enter)) 
{
    
    if(Window.IsBorderless == false)
    {
        
        Window.IsBorderless = true;
        
    }
    else
    {

        Window.IsBorderless = false;

    }
   
}

oldState = newState;

``` 
As I said, the above code works fine, But I have replaced it with a ternary operation and now I'm wondering if the two statements withing the curly brackets can somehow be combined into one? 

```c#
if(oldState.IsKeyUp(Keys.Enter) && newState.IsKeyDown(Keys.Enter)) 
{
    
    bool windowState = (Window.IsBorderless == false) ? Window.IsBorderless = true : Window.IsBorderless = false;

    Window.IsBorderless = windowState;  

}
``` 
Looking forward to seeing your feedback.....

**Comment by KelsamGames** [comment]
`Window.IsBorderless = !Window.IsBorderless;`

**Reply by BugInfestedCode** [reply]
> `Window.IsBorderless = !Window.IsBorderless;`

Thank you, KelsamGames. That's perfect !!! 

I looked at the code earlier and realised that I could do the following, but your solution is far better.

>`Window.IsBorderless = (Window.IsBorderless == false ? Window.IsBorderless = true : Window.IsBorderless = false);`

**Comment by Apostolique** [comment]
This is the way that I do it: <https://learn-monogame.github.io/how-to/fullscreen/>.
